fontforge/python.c

```diff
@@ -810,6 +810,114 @@ static PyObject *PyFF_UnicodeNamesListVersion(PyObject *UNUSED(self), PyObject *
     return( ret );
 }
 
+/* Ligature & Fraction information based on current Unicode (builtin) chart. */
+/* Unicode chart seems to distinguish vulgar fractions from other fractions. */
+/* These routines test value with internal table. Returns true/false values. */
```

Comment:
I think that it would be worthwhile to specify that the argument to each of these new functions is a numeric codepoint. It is not obvious.